test/test_site.py

```diff
@@ -91,7 +91,8 @@ def test_update(self):
             single_site = TSC.SiteItem(name='Tableau', content_url='tableau',
                                        admin_mode=TSC.SiteItem.AdminMode.ContentAndUsers,
                                        user_quota=15, storage_quota=1000,
-                                       disable_subscriptions=True, revision_history_enabled=False)
+                                       disable_subscriptions=True, revision_history_enabled=False,
+                                       materialized_views_enabled=False)
```

Comment:
Could you add 'materializedViewsEnabled' to the server response xml (test/assets/site_update.xml) and verify that we are parsing the response properly?
Also would be nice to have test_workbook.py updated, since the new field is added to workbook_item.